library-client-rtsp/src/main/java/com/alexvas/rtsp/widget/RtspSurfaceView.kt

```diff
@@ -30,12 +30,12 @@ open class RtspSurfaceView: SurfaceView {
     private var requestVideo = true
     private var requestAudio = true
     private var rtspThread: RtspThread? = null
-    private var videoFrameQueue = FrameQueue(60)
+    var videoFrameQueue = FrameQueue(200)
```

Comment:
Why queue set to 200? Why it is not private?

Reply:
The queue was filling up rapidly due to a combination of a long-running stream with a high frame rate and an insufficient FrameQueue size of 60. This size is adequate for streams with lower frame rates, but it becomes a bottleneck for streams that send frames more frequently.